nautobot_floor_plan/templates/nautobot_floor_plan/floorplan_retrieve.html

```diff
@@ -41,6 +42,14 @@
                 <td>Y Axis Labels</td>
                 <td>{{ object.y_axis_labels }}</td>
             </tr>
+            <tr>
+                <td>X Axis Seed</td>
+                <td>{{ object|seed_conversion:'x' }}</td>
+            </tr>
+            <tr>
+                <td>Y Axis Seed</td>
+                <td>{{ object|seed_conversion:'y' }}</td>
+            </tr>
```

Comment:
```suggestion
                <td>X Axis Seed</td>
                <td>{{ object|seed_conversion:'x' }}</td>
            </tr>
            <tr>
                <td>Y Axis Labels</td>
                <td>{{ object.y_axis_labels }}</td>
            </tr>
            <tr>
                <td>Y Axis Seed</td>
                <td>{{ object|seed_conversion:'y' }}</td>
            </tr>
```

Same as previous, grouping X/X and Y/Y